x-pack/plugin/sql/qa/src/main/resources/time.csv-spec

```diff
@@ -1,26 +1,26 @@
 //
 // TIME
 //
+// All tests are run with UTC timezone
```

Comment:
Is there a GH issue for the timezone problem with Time data type? If so, I would add it here and in the `CsvSpecTestCase` class as a TODO.

Reply:
No, I can create one